include/RAJA/RAJA.hxx

```diff
@@ -118,26 +113,26 @@
 //
 // All platforms must support sequential execution.
 //
-#include "RAJA/exec-sequential/raja_sequential.hxx"
+#include "RAJA/sequential.hxx"
 
 //
 // All platforms should support simd execution.
 //
-#include "RAJA/exec-simd/raja_simd.hxx"
+#include "RAJA/simd.hxx"
 
 #if defined(RAJA_ENABLE_CUDA)
-#include "RAJA/exec-cuda/raja_cuda.hxx"
+#include "RAJA/cuda.hxx"
 #endif
 
 #if defined(RAJA_ENABLE_OPENMP)
-#include "RAJA/exec-openmp/raja_openmp.hxx"
+#include "RAJA/openmp.hxx"
 #endif
 
 #if defined(RAJA_ENABLE_CILK)
-#include "RAJA/exec-cilk/raja_cilk.hxx"
+#include "RAJA/cilk.hxx"
 #endif
 
-#include "RAJA/IndexSetUtils.hxx"
+#include "RAJA/internal/IndexSetUtils.hxx"
```

Comment:
There's not much there now, but this stuff could be useful to end users.

Also, it seems a bit odd to me that we have "internal" stuff in the main header file.